requirements.txt

```
@@ -17,7 +18,7 @@ fast-diff-match-patch==2.1.0
 typing-extensions==4.9.0 

```

```suggestion
typing-extensions==4.9.0
```

Also delete the spaces here.